Alexey/Tax Calculator.py

```diff
@@ -0,0 +1,14 @@
+def main():

```

Comment:
It doesn't work correctly 
Test:
Type number:120
Type number:50
180.0
Expected:
240